common/src/main/java/tachyon/Constants.java

```diff
@@ -291,9 +291,9 @@
 
   public static final String S3_ACCESS_KEY = "fs.s3n.awsAccessKeyId";
   public static final String S3_SECRET_KEY = "fs.s3n.awsSecretAccessKey";
-  public static final String S3_PROXY_HOST = "fs.s3n.proxy-host";
-  public static final String S3_PROXY_PORT = "fs.s3n.proxy-port";
-  public static final String S3_PROTOCOL = "fs.s3n.protocol";
+  public static final String UNDERFS_S3_PROXY_HOST = "tachyon.underfs.s3.proxy.host";
+  public static final String UNDERFS_S3_PROXY_PORT = "tachyon.underfs.s3.proxy.port";
+  public static final String UNDERFS_S3_PROXY_HTTPS_ONLY = "tachyon.underfs.s3.proxy.https.only";
```

Comment:
@aaudiber could you also add the fields to `docs/Configuration-Settings.md`? Also, could you move the String constants to the `tachyon.underfs` area (top of file)?